salt/states/pip_state.py

```diff
@@ -1065,3 +1065,51 @@ def uptodate(name,
         ret['comment'] = 'Upgrade failed.'
 
     return ret
+
+
+def mod_aggregate(low, chunks, running):
+    '''
+    The mod_aggregate function which looks up all packages in the available
+    low chunks and merges them into a single pkgs ref in the present low data
+    '''
```

Comment:
Some docs here for the args... unless you wanna make it a hidden function. (`_mod_aggregate`)

Reply:
This is the same doc string that exists in the iptables and pkg state modules, it's not a function that is intended to be called directly but since it's called outside of the state module it can't be a private function.